evm/utils/db.py

```diff
@@ -8,42 +8,50 @@
     EMPTY_SHA3,
 )
 
+from evm.rlp.blocks import BaseBlock
+from evm.rlp.headers import BlockHeader
 
-def make_block_number_to_hash_lookup_key(block_number):
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from evm.db.chain import BaseChainDB  # noqa
```

Comment:
Can you add the specific error we need to ignore to the `noqa` comment.

Reply:
Good point. Will change.